users/residents.md

```diff
@@ -0,0 +1,9 @@
+
```

Comment:
I'd remove this empty rule at the beginning of the document (and add one after the title)